src/client/styles/main.scss

```diff
@@ -170,3 +170,66 @@ input[type=submit]:focus {
   outline-style: solid;
   outline-width: 3px;
 }
+
+.nypl-results-list {
```

Comment:
Looks good. But the CSS will eventually go to TK? We just have it here a temporary solution?